rclcpp/include/rclcpp/node.hpp

```diff
@@ -74,46 +82,46 @@ class Node
     std::string topic_name,
     size_t queue_size,
     std::function<void(const std::shared_ptr<MessageT> &)> callback,
-    rclcpp::callback_group::CallbackGroup::SharedPtr group=nullptr);
+    rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr);
 
   /* Create a timer. */
   rclcpp::timer::WallTimer::SharedPtr
   create_wall_timer(
     std::chrono::nanoseconds period,
     rclcpp::timer::CallbackType callback,
-    rclcpp::callback_group::CallbackGroup::SharedPtr group=nullptr);
+    rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr);
 
   rclcpp::timer::WallTimer::SharedPtr
   create_wall_timer(
     std::chrono::duration<long double, std::nano> period,
     rclcpp::timer::CallbackType callback,
-    rclcpp::callback_group::CallbackGroup::SharedPtr group=nullptr);
+    rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr);
 
   typedef rclcpp::callback_group::CallbackGroup CallbackGroup;
   typedef std::weak_ptr<CallbackGroup> CallbackGroupWeakPtr;
   typedef std::list<CallbackGroupWeakPtr> CallbackGroupWeakPtrList;
 
   /* Create and return a Client. */
-  template <typename ServiceT>
+  template<typename ServiceT>
   typename rclcpp::client::Client<ServiceT>::SharedPtr
   create_client(
     std::string service_name,
-    rclcpp::callback_group::CallbackGroup::SharedPtr group=nullptr);
+    rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr);
 
   /* Create and return a Service. */
-  template <typename ServiceT>
+  template<typename ServiceT>
   typename rclcpp::service::Service<ServiceT>::SharedPtr
   create_service(
     std::string service_name,
     std::function<void(const std::shared_ptr<typename ServiceT::Request> &,
-                       std::shared_ptr<typename ServiceT::Response>&)> callback,
-    rclcpp::callback_group::CallbackGroup::SharedPtr group=nullptr);
+    std::shared_ptr<typename ServiceT::Response> &)> callback,
+    rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr);
```

Comment:
uncrustify does not change the way the first argument is positioned. It only decides the indentation of the wrapped arguments here. We have to manually wrap the first argument in this case.